pymeasure/instruments/agilent/agilentL4534A.py

```diff
@@ -0,0 +1,425 @@
+            values=FILTER_VALUES
+        )
+
+        def _read_data_block(self, dtype):
+            """
+            Read data block header response that some FETCh commands return by default
+            :param dtype: Numpy data type expected for this data block
+            :return: numpy array of specified type
+            """
+            # Data block has header of form '#<n><n digits>',
+            # where <n>specifies the number of digits used to indicate the size of the block,
+            # and the <n digits> indicate the block size in bytes.
+            digits = self.read_bytes(2).decode()
+            if digits.startswith('#'):
+                try:
+                    bytes = int(self.read_bytes(int(digits[1])).decode())
+                    # Read specified number of bytes + newline terminator
+                    buf = memoryview(self.read_bytes(bytes+1))
+                    # Return view that strips off newline to contain only desired bytes
+                    return np.frombuffer(buf[:-1], dtype)
+                except ValueError:
+                    InstrumentError(
+                        """
+                        Unable to parse data block header returned by instrument.
+                        """
+                    )
+            else:
+                raise InstrumentError(
+                    """
+                    Data block header character "#" not found in response.
+                    Is device in ASCII mode?
+                    """
+                    )
+
+        @property
+        def voltage(self):
+            """Get voltage measurements for this channel (array of float in V)."""
+            self.write(f'FETC:WAV:VOLT? (@{self.id})')
+            return ureg.Quantity(self._read_data_block('>f4').astype(np.float32), ureg.V)
+
+        @property
+        def adc(self):
+            """
+            Get raw ADC measurements for this channel in counts (int from -32767 to +32767)
+            in current voltage range.
+            """
+            self.write(f'FETC:WAV:ADC? (@{self.id})')
+            return self._read_data_block('>i2').astype(np.int16)
+
+    channels = Instrument.ChannelCreator(DigitizerChannel, (1, 2, 3, 4))
+
+    display = Instrument.control(
+        'DISP:TEXT?',
+        'DISP:TEXT \"%s\"',
+        """Control Display text on screen, up to 12 characters.""",
+        get_process=lambda v: v.strip('\"')
+    )
+
+    def clear_display(self):
+        return self.write('DISP:TEXT:CLE')
+
+    arm_source = Instrument.control(
+        "CONF:ARM:SOUR?",
+        "CONF:ARM:SOUR %s",
+        """Set the source used to arm the digitizer (IMMediate|EXTernal|SOFTware|TIMer).""",
+        validator=strict_discrete_set,
+        values=['IMM', 'SOFT', 'EXT', 'TIM']
+    )
+
+    ext_slope = Instrument.control(
+        "CONF:EXT:INP?",
+        "CONF:EXT:INP %s",
+        """Set the edge to be used for the external trigger input (NEGative or POSitive).""",
```

Comment:
It is a control, so the docstring should start with "Control", not "Set". Unless it is a "setting" (not readable).